modules/test/playwright/tests/commerce/commerce-order-web/main/commerceAdminOrder.spec.ts

```diff
@@ -1175,3 +1175,25 @@ test(
 		});
 	}
 );
+
+test(
```

Comment:
This looks like a little overkill. Can't we just integrate this check in whatever current Minium set-up (as we do need it), instead of redoing everything from scratch just to validate the presence of a label?